plugins/hls-call-hierarchy-plugin/src/Ide/Plugin/CallHierarchy/Internal.hs

```diff
@@ -115,7 +116,9 @@ construct nfp hf (ident, contexts, ssp)
         renderSpan _           = ssp
 
         -- https://github.com/haskell/lsp/blob/e11b7c09658610f6d815d04db08a64e7cf6b4467/lsp-types/src/Language/LSP/Types/DocumentSymbol.hs#L97
-        skUnknown = SkUnknown 27 -- 27 is the first unused number while ToJSON
+        -- There is no longer an unknown symbol, thus using SymbolKind_TypeParameter
```

Comment:
Given it's a call hierarchy I'd be inclined to use "function" for an unknown item in the hierachy?